var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -57,3 +58,27 @@ class PyLibensemble(PythonPackage):
     depends_on('py-deap', type=('build', 'run'), when='+deap')
     depends_on('tasmanian+python', type=('build', 'run'), when='+tasmanian')
     conflicts('~mpi', when='@:0.4.1')
+
+    @run_after('install')
+    def cache_test_sources(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(join_path('examples', 'tutorials'))
+
+    def run_tutorial_tests(self, exe):
+        """Run tutorials stand alone test"""
+
+        test_dir = join_path(self.test_suite.current_test_cache_dir,
+                             'examples', 'tutorials')
+
+        if not os.path.exists(test_dir):
+            print('Skipping {0} test'.format(exe))
+            return
+
+        self.run_test(self.spec['python'].command.path,
+                      options=[exe, '--comms', 'local', '--nworker', '1'],
```

Comment:
The option is still plural regardless of the number of workers provided here.

```suggestion
                      options=[exe, '--comms', 'local', '--nworkers', '1'],
```